src/Raven.Client/Documents/Session/Tokens/GroupByKeyToken.cs

```diff
@@ -27,7 +27,7 @@ public override void WriteTo(StringBuilder writer)
 
             writer
                 .Append(" as ")
-                .Append(_projectedName);
+                .Append($"\'{_projectedName}\'");
```

Comment:
What is the name also contains `'`?

Reply:
I do not think we are doing that anywhere.